kotlin/internal/opts.bzl

```diff
@@ -49,6 +49,13 @@ _KOPTS = {
         ),
         type = attr.bool,
     ),
+    "x_opt_in": struct(
+        args = dict(
+            default = [],
+            doc = "opt in to experimental features",
```

Comment:
Hmmmm... https://kotlinlang.org/docs/reference/opt-in-requirements.html

This looks like it's going to be a user definable configuration over time. Could you open an issue for deciding the direction of the opt in api? 

If there are specific usecases you need to unblock, we can work around the larger design considerations for now.

Reply:
Good point - it IS possible for library writers to define something needs to be opt in. That + the need for experimental coroutine apis (which are sadly still very common) are why I figured it be possible to define this. Ill open the issue but would love for this to be added soon as a lot of APIs are unavailable for use until this flag is here.